skepticoin/networking/peer.py

```diff
@@ -257,6 +242,9 @@ def add_transaction_to_pool(self, transaction):
 
             try:
                 validate_non_coinbase_transaction_by_itself(transaction)
+
+                assert self.coinstate
```

Comment:
as elsewhere

Reply:
Resolving this one, see other thread about this topic.